cmd/provider/main.go

```diff
@@ -59,7 +59,7 @@ func main() {
 		leaderRenew    = app.Flag("leader-renew", "Set leader election renewal.").Short('r').Default("10s").OverrideDefaultFromEnvar("LEADER_ELECTION_RENEW").Duration()
 
 		syncInterval         = app.Flag("sync", "How often all resources will be double-checked for drift from the desired state.").Short('s').Default("1h").Duration()
-		pollInterval         = app.Flag("poll", "How often individual resources will be checked for drift from the desired state").Short('p').Default("1m").Duration()
+		pollInterval         = app.Flag("poll", "How often individual resources will be checked for drift from the desired state").Short('p').Default("30m").Duration()
```

Comment:
I may be misunderstanding but does this mean we'll take up to 30 minutes to react to a deletion?

Reply:
No, we still reconcile kubernetes events, this means how often we check the status on the backends.